pkg/release/release.go

```diff
@@ -159,6 +159,11 @@ func ensureMetadata(downloadPath, metadataFileName string) (Metadata, error) {
 func (r *Release) CheckHelmCharts() error {
 	// check if the cluster class chart is present.
 	clusterClassChartName := r.clusterClassChartName()
+
+	if _, err := r.ClusterClassChartPath(); err != nil {
```

Comment:
isn't this the same as the one below?

Reply:
> isn't this the same as the one below?
yes, indeed; I removed it

Reply:
After some refactoring the tests are green now and everything looks good to go again.